corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    name.capitalize!
+    name[0...10] if name.length > 10
+  end
```

Comment:
- [ ] Hi @vic778 , from the requirements of this project, we should equally have a `Decorator` class that inherits from your `Corrector(Nameable)` class. Please let's include this class, I guess it will be fun. 
- [ ] Also, we shall require a `CapitalizeDecorator `class that will inherit from the `Decorator` class and should contain a `correct_name` method that capitalized the first letter of the given name. It should look something similar like this: 

```ruby
class CapitalizeDecorator < Decorator
    def correct_name
      @nameable.correct_name.capitalize
    end
end
```

- [ ] Once more, from the requirements of this project, you should implement `TrimmerDecorator `class that should inherit from the same `Decorator` class and should contain a` correct_name` method which makes sure that the name should have a maximum of 10 characters else it should trim the word.